src/ex1/index.html

```diff
@@ -1,14 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <title>Exercise 1</title>
-    <link rel="stylesheet" href="style.css" />
+  <title>TODO RAVE</title>
+  <link rel="stylesheet" href="style.css" />
 </head>
-<body>
 
-<h1>Todo list</h1>
+<body>
+  <div id="myDIV" class="header">
+    <h2 class="Heading">NBA Legands To Do List</h2>
 
+    <div class="container" style="margin-bottom: 10px; visibility: hidden">
```

Comment:
having inline style is usually not a best practice. in this case you already have a css class name, why not putting the style inside it?